include/rsg/services/globalService.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef _RSG_GLOBAL_SERVICE_IMPL_
```

Comment:
I seem to understand that subsequent commits get ride of that file (I didn't saw the end of the film yet), but I have the feeling that when you are saying `global` you mean `engine` to mimick the simgrid parlance.